### ISSUE TITLE
update

### ISSUE THREAD

**Issue by Fre68** [WWBN/AVideo#8009]
Good morning
I want to update Avideo, but I understand absolutely nothing about the process. Git pull and other indications are foreign to me.
Isn't there a way to update everything by clicking on a button update? Or by a plugin?
I'm afraid of breaking everything

Frédéric

**Comment by DanielnetoDotCom** [comment]
no, update AVideo requires from you a minimum command line knowledge

**Comment by LATINMIXMASTERS** [comment]
I will suggest to pay the service charge with Daniel is worth it.
